README.md

```diff
@@ -43,6 +42,9 @@ Source this file in and the environment is ready for using the connector with OA
 $ source .env
 ```
 
+#### Callback URI
+The OAuth flow requires that the authorizing server can call back to the initiating client with an authorization code. When in OAuth mode, the connector will start a transient web server that by default will listen for an authorization code on `http://localhost:5000/oauth2/callback`. This can be modified by setting the appropriate value for `VISIER_REDIRECT_URI`. This value has to match the value provided during client registration exactly. In addition, it must abide by Visier's callback URI rules such as a limited set of permissable subnets. Consult your Visier documentation for details around acceptable callback URIs.
```

Comment:
The OAuth flow requires that the authorizing server can call back to the initiating client with an authorization code. In OAuth mode, the connector starts a transient web server that listens for an authorization code on `http://localhost:5000/oauth2/callback`. You can modify the URL by setting a different value for `VISIER_REDIRECT_URI`. The `VISIER_DIRECT_URI` value must exactly match the URI value in your Visier OAuth 2.0 client registration and must abide by Visier's callback URI rules, such as a limited set of permissible subnets.